### PR TITLE
v1/consumer: Add consumer API

### PR DESCRIPTION
**Pull request by nkristianto**
Add consumer API definition.

### CODE REVIEW

File: v1/consumer/consumer.gunk [hunk 1]
```diff
@@ -0,0 +1,393 @@
...
+type CreateByUser struct {
+	UserID     string `pb:"1" json:"user_id"`
+	Email      string `pb:"2" json:"email"`
+	ProviderID string `pb:"3" json:"provider_id"`
+	Provider   string `pb:"4" json:"provider"`
+	Username   string `pb:"5" json:"username"`
+}
+
+type Consumer struct {
+	ConsumerID      int          `pb:"1" json:"consumer_id"`
```
Comment (ghazninattarshah) [inline v1/consumer/consumer.gunk]: may be just `ID`

File: v1/consumer/consumer.gunk [hunk 2]
```diff
@@ -0,0 +1,393 @@
...
+	//         Method: "GET",
+	//         Path:   "/v1/management/consumers/{ConsumerID}/consumer/call-limits",
+	// }
+	// +gunk openapiv2.Operation{
+	//         Tags:        []string{"Consumer"},
+	//         Description: "Returns a list up to 20 consumer call limits data.",
+	//         Summary:     "Get consumer call lists",
+	//         Responses: map[string]openapiv2.Response{
+	//                 "200": openapiv2.Response{
+	//                         Description: "Request executed successfully.",
```
Comment (ghazninattarshah) [inline v1/consumer/consumer.gunk:214]: missing schema tag here. 

``` Schema: openapiv2.Schema{
	//                                 JSONSchema: openapiv2.JSONSchema{
	//                                         Ref: "#/definitions/consumerGetConsumerCallLimitsResponse",
	//                                 },
	//                         }
```

File: v1/consumer/consumer.gunk [hunk 3]
```diff
@@ -0,0 +1,393 @@
...
+	//         Method: "GET",
+	//         Path:   "/v1/management/consumers/{ConsumerID}",
+	// }
+	// +gunk openapiv2.Operation{
+	//         Tags:        []string{"Consumer"},
+	//         Description: "Returns consumer data by it's ID",
+	//         Summary:     "Get consumer",
+	//         Responses: map[string]openapiv2.Response{
+	//                 "200": openapiv2.Response{
+	//                         Description: "Request executed successfully.",
```
Comment (ghazninattarshah) [inline v1/consumer/consumer.gunk:248]: missing schema tag here.

**Comment by ghazninattarshah** [comment]
Missing schema tag for various func.

**Review by ghazninattarshah — CHANGES_REQUESTED** [review]
LGTM with minor changes.

File: v1/consumer/consumer.gunk (hunk 1 above)
Comment (nkristianto) [reply v1/consumer/consumer.gunk]: thanks

File: v1/consumer/consumer.gunk (hunk 2 above)
Comment (nkristianto) [reply v1/consumer/consumer.gunk:214]: do we need to use schema tag for get request?

**Review by ghazninattarshah — APPROVED** [review]
LGTM

**Pull request merged**